pkg/scheduler/server/api.go

```diff
@@ -145,6 +146,7 @@ func (s *Server) WatchJobs(stream schedulerv1pb.Scheduler_WatchJobsServer) error
 	}
 
 	s.connectionPool.Add(req.GetInitial(), stream)
+	monitoring.RecordSidecarsConnectedCount()
```

Comment:
```suggestion
	s.cnt.Add(1)
	monitoring.RecordSidecarsConnectedCount(s.cnt.Load())

	defer func() {
		s.cnt.Add(-1)
		monitoring.RecordSidecarsConnectedCount(s.cnt.Load())
	}()
```
Assuming `cnt atomic.Int64`